test/e2e/baseline_test.go

```diff
@@ -610,6 +612,77 @@ func TestMixNsAndServiceWaypoint(t *testing.T) {
 	})
 }
 
+func TestBookinfo(t *testing.T) {
+	framework.NewTest(t).Run(func(t framework.TestContext) {
+		namespace := apps.Namespace.Name()
+		// Install bookinfo.
+		if _, err := shell.Execute(true, fmt.Sprintf("kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml -n %s", namespace)); err != nil {
+			t.Fatalf("failed to install bookinfo: %v", err)
+		}
+		t.Cleanup(func() {
+			if _, err := shell.Execute(true, fmt.Sprintf("kubectl delete -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml -n %s", namespace)); err != nil {
```

Comment:
Does this need to change with istio version changes

Reply:
> Does this need to change with istio version changes

It should not be needed at present. We only use very basic functions here and bookinfo itself will not change much with upgrade of istio.